packages/carbon-graphs/dev/app.js

```diff
@@ -0,0 +1,1194 @@
+
```

Comment:
What is this 'dev' folder used for?

Reply:
The dev folder hosts the examples used in the carbon dev-site.